src/main/graphql/GetConsignmentExport.graphql

```diff
@@ -0,0 +1,14 @@
+query getConsignmentExport($consignmentId: UUID!) {
+    getConsignment(consignmentid: $consignmentId) {
+        userid
```

Comment:
I already commented on `datetime` in https://github.com/nationalarchives/tdr-consignment-api/pull/115, but I've just noticed that `userid` is similar - could it be `createdBy` or something like that? We'll eventually have a `lastModifiedBy` user ID too.

Reply:
We could change `userid` to `createdBy`, but would this actually make sense, given the value would be a UUID? Maybe `createdByUuid` would be better?

Reply:
I suppose the GraphQL answer would be to give it more structure:

```
createdBy {
  id
}
```

where the GraphQL type of `createdBy` is a `User` type, which leaves open the possibility of returning more information later. Not sure if that's a bit excessive.

Since writing the original comment, though, I notice that `userid` was already returned by the `getConsignment` endpoint. Does that mean something else was already using it?

Reply:
The userid field from the Consignment object is definitely used in the consignment-api for authorisation. Whether any repo using the generated-graphql makes use of this field I'm not sure

Reply:
As discussed on Slack, the userid field isn't new, so it will take some care to rename or restructure it. So let's leave it for now, and revisit it later, perhaps when we rename the field in the DB.